modules/mapbox/src/deck-utils.ts

```diff
@@ -300,7 +300,12 @@ type MaplibreRenderParameters = {
 
 function getViewport(deck: Deck, map: Map, renderParameters?: unknown): Viewport {
   const viewState = getViewState(map);
-  const view = getDefaultView(map);
+  const {
+    props: {views}
+  } = deck;
+  const view =
+    (views && flatten(views).filter((v: {id: string}) => v.id === 'mapbox')[0]) ||
```

Comment:
This raking for `'mapbox'` feels hacky, like it deserves a comment?

Reply:
It's probably a good time to pull it out into a const. The magic string is [documented](http://deck.gl/docs/api-reference/mapbox/mapbox-overlay#multi-view-usage) - everything I'd say in a comment is there